sdk/ai/azure-ai-projects/azure/ai/projects/models/_patch.py

```diff
@@ -991,6 +997,94 @@ def until_done(self) -> None:
             pass
 
 
+class Messages:
```

Comment:
One of the frequent comments in SDK reviews is about one-work class names, that are very generic.  Can we make this a two-word class name to be more specific?

Reply:
I try to get Johan comments, also will think about more specific name

Reply:
Changed to ThreadMessages